src/sentry/models/team.py

```diff
@@ -250,3 +270,12 @@ def get_projects(self):
         from sentry.models import Project
 
         return Project.objects.get_for_team_ids({self.id})
+
+    def delete(self, **kwargs):
+        from sentry.models import Actor, ExternalActor
+
+        # There is no foreign key relationship so we have to manually cascade.
+        ExternalActor.objects.filter(actor=self.actor).delete()
+        Actor.objects.filter(id=self.actor_id).delete()
```

Comment:
The `ExternalActor` should get deleted when the `Actor` is deleted right? So do we need both of these here? can we just keep `Actor.objects.filter(id=self.actor_id).delete()`?